spec/rspec/core/example_spec.rb

```diff
@@ -58,6 +58,21 @@ def metadata_hash(*args)
     end
   end
 
+  it "captures example timing even for exceptions unhandled by RSpec" do
+    begin
+      avoided = RSpec::Support::AllExceptionsExceptOnesWeMustNotRescue::AVOID_RESCUING
+      unhandled = Class.new(StandardError)
+      avoided << unhandled
```

Comment:
I'd be ok with `OutOfMemory` -- or you could consider this:

``` ruby
unhandled = RSpec::Support::AllExceptionsExceptOnesWeMustNotRescue::AVOID_RESCUING.first
```

Basically, just grab the first one, without caring about what it is.  After all, we don't care about what it is, and this makes it clear that we are just using one of the unhandled exceptions, not that the specific unhandled exception is important.